### DISCUSSION TITLE
after fresh install ERR_TOO_MANY_REDIRECTS

### DISCUSSION THREAD

**Discussion by FidelVe** [modoboa/modoboa#1862] [Help]
after a fresh install, I try to enter mail.website.com and Chrome and Chromium browser shows

ERR_TOO_MANY_REDIRECTS

this is even after clearing cookies
![nginx](https://user-images.githubusercontent.com/43641727/78119179-060e0880-73d6-11ea-9f01-9eaf085835da.png)

# Impacted versions

* Modoboa: latest I guess dont know how to check for the version
* installer used: Yes
* Webserver: Nginx

# Steps to reproduce
fresh install on ubuntu 18.04

# Current behavior

# Expected behavior

# Video/Screenshot link (optional)

**Comment by Paultje52** [comment]
Hello.
I had the exact same issue, but I found a solution if you're using cloudflare and you have SSL to flexible. What you need to do is go to your cloudflare settings, create a page rule for `mail.domain.com/*`, and set SSL to full.
